CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines
+
+## Legal
+
+Contributions must be all your own work; plagiarism is not allowed.
+By submitting a pull request, you agree to license your contribution
+under the [license of this repository](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/LICENSE.md).
+
+## Translations
+
+Translations go in the appropriate folder named by the locale code.
+
+Do not change the structure or formatting of the original (English) explanation when translating or updating translations.
+You may change number formatting to fit the locale
+(`42.33` may be formatted as `42,33` in a german translation for instance)
+as long as it doesn't create ambiguities
+(e.g. `[42,33, 13]` in an array would be disallowed and should either remain `[42.33, 13]` or use a different delimiter `[42,33; 13]`).
+
+## Writing Explanations
+
+### Structure
```

Comment:
To make it easier for people to add an algorithm explanation, we could add a typical structure of how it should look. We can remove it from the root `README.md` of each directory.

Reply:
I'd use #201 for that.

Reply:
We can use that. 🙂